pytorch_lightning/utilities/enums.py

```diff
@@ -66,6 +66,7 @@ class DistributedType(LightningEnum):
     HOROVOD = 'horovod'
     DDP_SHARDED = 'ddp_sharded'
     DDP_SHARDED_SPAWN = 'ddp_sharded_spawn'
+    FULL_SHARDED = 'full_sharded'
```

Comment:
but still, it is DDP, right?

Reply:
yes, I think it should be `DDP_FULL_SHARDED`

Reply:
DDP_FULLY_SHARDED or DDP_FULL_SHARDS. It always a bit bothered me that "shard" is a noun. :-)

Reply:
Let's go with `DDP_FULLY_SHARDED` and I'll update everything!